app/views/layouts/mailer.html.haml

```diff
@@ -0,0 +1,3 @@
+%hmtl
+  %body
+    = yield
```

Comment:
This is a very minor one, GitHub complains about the missing new line at the end of the file. This is a generated file, that's why there is no new line at the end I guess, otherwise TextMate adds it by default I guess. Feel free to ignore this suggestion.